CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+# 1.0.9-rc1 (2019-08-09)
+* Support running the Windows installer script in 32-bit PS host on 64-bit Windows ([76eacaf](https://github.com/Azure/iotedge/commit/76eacaf988e7d421b4e45c70a45d8160f1feaaa1))
+* `iotedge check` improvements ([b5b2352](https://github.com/Azure/iotedge/commit/b5b2352f78e9d18b4b3f301ea42e3b5ded85fb6d))
```

Comment:
Probably more useful to link to 8090d6c7fc47a377787d5d9ff15ed5fea894112c since that adds `iotedge check-list`, `iotedge check --dont-run` and `iotedge check --warnings-as-errors`